flake8_print.py

```diff
@@ -1,54 +1,34 @@
-import ast
+"""Extension for flake8 that finds print invocations."""
 import tokenize
 
-from sys import stdin
-
-__version__ = '1.6.1'
+__version__ = '1.7.0'
 
 PRINT_ERROR_CODE = 'T001'
 PRINT_ERROR_MESSAGE = 'print statement found.'
 
-
-class PrintStatementChecker(object):
```

Comment:
This could be re-added, either by
1. wrapping pep8 Checker as the test module does, or 
2. manually tokenizing the stream.  However, 'logical_lines' and 'noqa' would be hard to implement properly.